client/src/modules/authorization/components/SignUpInput.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames';
+import _debounce from 'lodash/debounce';
+import _trim from 'lodash/trim';
+import _trimStart from 'lodash/trimStart';
+
+import { IntlPropType } from 'common/constants/propTypes';
+import { CheckIcon, ErrorIcon } from 'assets/images/icons';
+
+class SignUpInput extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      errorMessageId: '',
+      invalidView: false,
```

Comment:
View is more like "widok", you mean "wygląd", so it would be better to use 
invalidTheme, validTheme